src/Standards/PSR12/Docs/Classes/ClosingBraceStandard.xml

```diff
@@ -0,0 +1,35 @@
+<documentation title="Closing Brace">
+    <standard>
+    <![CDATA[
+    Any closing brace must not be followed by any comment or statement on the same line.
```

Comment:
Kudos on using the PSR12 phrasing. The thing I'm wondering about is whether this, on its own, is clear enough.

In the PSR12 specs this rule is part of the "Classes, Properties, and Methods" chapter and the sniff follows that and only applies the rule to OO structures (classes, interfaces, traits, enums) and functions/methods.

As this phrase is now used without that additional context, the "any closing brace" could be interpreted to include control structure closing braces and arbitrary closing braces, i.e. `{$var}`, while the sniff does not apply to those.

Reply:
So should I just add the remark to the effect of:

> In the context of object-oriented structures (classes, interfaces, methods), any closing brace must not be followed by any comment, or statement on the same line.

I was also thinking of using the phrase `class-like constructs` that would encompass classes, interfaces, traits and enums, instead of listing some of them:

> In the context of object-oriented structures (class-like constructs, methods), any closing brace must not be followed by any comment, or statement on the same line.

From the documentation point of view, the more explanation we add, the better. But I always thought that the sniff docs should be smaller and concise.

Reply:
Been pondering a little on the phrasing. What about this ?
```
The closing brace of OO constructs and functions must not be followed by any comment or statement on the same line.
```

Reply:
Sounds good to me, will change it, but would leave the `object-oriented` instead of OO for a bit more clarity.